### PR TITLE
more readme - about and support

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

**Review by eddynaka — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,7 +4,29 @@
...
+This repository is maintained by [.NET SDK
+maintainers](https://github.com/orgs/open-telemetry/teams/dotnet-maintainers)
+team and [.NET SDK
+approvers](https://github.com/orgs/open-telemetry/teams/dotnet-approvers) can
+help with reviews and code approval. However, as individual components are
+developed by numerous contributors, approvers and maintainers are not expected
+to directly contribute to every component.
+
+This repository is great for community supported components. Vendor specific
+code that deliver a higher supportability guarantees needs to be placed in
```
Comment (CodeBlanch) [inline README.md:30]: Something about `deliver a higher supportability guarantees` seems a bit off.

`requires a higher supportability guarantee`

Maybe?

Comment (SergeyKanzhelev) [reply README.md]: fixed

**Comment by SergeyKanzhelev** [comment]
mention: https://github.com/open-telemetry/opentelemetry-dotnet/issues/1030

**Review by CodeBlanch — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (CodeBlanch) [reply README.md]: I think it should also be `guarantee` not `guarantees`?

**Pull request merged**